src/apps/replication/replication.thrift

```diff
@@ -232,6 +268,7 @@ struct configuration_query_by_node_response
 struct configuration_query_by_index_request
 {
     1:string           app_name;
+    2:bool             if_query_all_partitions;
```

Comment:
how about if (partition_indices.size() == 0) then we are querying all partitions?